### DISCUSSION TITLE
Find line number for error

### DISCUSSION THREAD

**Discussion by psheehymn** [gapitio/gapit-htmlgraphics-panel#171] [Q&A] [answered]
When I am getting a JavaScript error from the HTML Graphics panel like "Cannot read properties of undefined (reading '0')" is there any way to isolate what line number the error is on? I've got 600+ LOC in my panel and I have read through it three times line-by-line and cannot determine the source of the error.

Thank you for any tips or tricks you may have.

**Comment by ZuperZee** [comment, marked as answer]
In the console (ctrl+shift+j) the error code usually has an offset of 2.
![image](https://github.com/gapitio/gapit-htmlgraphics-panel/assets/39485579/69f4ba8c-62aa-4792-85f1-892532d641fe)

**Reply by psheehymn** [reply]
Thank you, that was exactly what I needed!
